lib/iam/models/IamDiff.rb

```diff
@@ -74,10 +77,13 @@ def self.attached(added, removed)
 
   def diff_string
     case @type
-    when UNMANAGED_POLICY
```

Comment:
These are just reordered to be in alphabetical order. The real interesting part starts with

```ruby
when POLICY_DOC
```